src/tests/java/picard/sam/testers/SamFileTester.java

```diff
@@ -340,4 +342,8 @@ private File createInputFile() {
     public SamReader getInput() {
         return samRecordSetBuilder.getSamReader();
     }
+
+    public void setRecordsNeedSorting(final boolean recordsNeedSorting) {
```

Comment:
Bringing in @tfenne 
![top-gun-switching-to-guns](https://cloud.githubusercontent.com/assets/846274/15542944/e6eaeb14-2260-11e6-82ca-e924621b71ed.gif)